src/config.js

```diff
@@ -235,15 +235,18 @@ export async function parseConfig() {
 				repos.forEach((name) => {
 					const files = parseFiles(group.files)
 					const repo = parseRepoName(name)
+					const branchSuffix = group.branchSuffix || 'default'
+					const key = `${ repo.uniqueName }|${ branchSuffix }`
 
-					if (result[repo.uniqueName] !== undefined) {
```

Comment:
The current code creates a map using the repo name as the key. This means you can have a maximum of one PR per repo